x-pack/plugins/maps/public/actions/store_actions.js

```diff
@@ -519,8 +537,9 @@ export function setJoinsForLayer(layer, joins) {
   };
 }
 
-export async function loadMetaResources(dispatch) {
-  const meta = await fetch(`${GIS_API_RELATIVE}/meta`);
-  const metaJson = await meta.json();
-  await dispatch(setMeta(metaJson));
-}
+export const resetLayerLoad = () => {
+  return {
+    type: RESET_LAYER_LOAD
+  };
```

Comment:
Did you mean to commit this? I see it came in under 'tmp commit'

Reply:
yes. this action was the lone action in the `ui_actions.js` file, which this PR is removing